nbs/src/tbats.ipynb

```diff
@@ -26,11 +26,11 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #57.            if k == 0: 
</pre><p>This would be better by initializing <code>mod = {'aic': np.inf}</code> and just computing <code>new_mod</code>in every loop. Or is it possible that all models return infinite aic?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/759/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<blockquote>is it possible that all models return infinite aic?</blockquote><p> </p><p>No, with the way we're computing the AIC, I don't think this is possible. Will implement the recommended change. </p>